lib/browser_collection.js

```diff
@@ -38,30 +38,34 @@ class BrowserCollection {
     return this.browsers.map((browser) => browser.serialize())
   }
 
-  calculateExitCode (results, singleRunBrowserNotCaptured, failOnEmptyTestSuite, failOnFailingTestSuite) {
+  calculateExitCode (results, singleRunBrowserNotCaptured, config) {
+    config = config || {}
     if (results.disconnected || singleRunBrowserNotCaptured) {
       return 1
-    } else if (results.success + results.failed === 0 && !failOnEmptyTestSuite) {
+    } else if (results.skipped && config.failOnPendingTests) {
```

Comment:
SInce there's a return statement in every `if` block, there's no need for `else`. This will make the code cleaner.

Reply:
done